src/etl/LoadBalancer.hpp

```diff
@@ -94,10 +93,13 @@ class LoadBalancer : public etlng::LoadBalancerInterface, LoadBalancerTag {
     std::uint32_t downloadRanges_ =
         kDEFAULT_DOWNLOAD_RANGES; /*< The number of markers to use when downloading initial ledger */
 
-    std::reference_wrapper<util::prometheus::HistogramInt> forwardedDurationHistogram_;
-    std::reference_wrapper<util::prometheus::CounterInt> forwardedRetryCounter_;
-    std::reference_wrapper<util::prometheus::CounterInt> cacheTriedCounter_;
-    std::reference_wrapper<util::prometheus::CounterInt> cacheMissCounter_;
+    struct ForwardingCounters {
+        std::reference_wrapper<util::prometheus::CounterInt> SuccessDuration;
```

Comment:
Clang-tidy will be upset about this naming 🔢

Reply:
Thanks, fixed